librz/core/cmd/cmd_analysis.c

```diff
@@ -8812,3 +8729,274 @@ RZ_IPI RzCmdStatus rz_analysis_class_vtable_lookup_handler(RzCore *core, int arg
+
+	if (argc > 1) {
+		l = (st32)rz_num_math(core->num, argv[1]);
+		if (l <= 0) {
+			RZ_LOG_ERROR("Invalid zero or negative arguments.\n");
+			return RZ_CMD_STATUS_ERROR;
+		}
+
+		count = l;
+		l *= 8;
+		if (l > obs) {
+			rz_core_block_size(core, l);
+		}
+	}
+
+	switch (state->mode) {
+	case RZ_OUTPUT_MODE_JSON:
+		core_analysis_bytes_json(core, core->block, core->blocksize, count, state->d.pj);
+		break;
+	case RZ_OUTPUT_MODE_STANDARD:
+		core_analysis_bytes_standard(core, core->block, core->blocksize, count);
+		break;
+	default:
+		rz_warn_if_reached();
+		break;
+	}
+
+	if (obs != core->blocksize) {
+		rz_core_block_size(core, obs);
+	}
+	return RZ_CMD_STATUS_OK;
+}
+
+RZ_IPI RzCmdStatus rz_analyze_n_ins_size_handler(RzCore *core, int argc, const char **argv) {
+	ut32 count = 1, obs = core->blocksize;
+	st32 l;
+
+	if (argc > 1) {
+		l = (st32)rz_num_math(core->num, argv[1]);
+		if (l <= 0) {
+			RZ_LOG_ERROR("Invalid zero or negative arguments.\n");
+			return RZ_CMD_STATUS_ERROR;
+		}
+		count = l;
+		l *= 8;
+		if (l > obs) {
+			rz_core_block_size(core, l);
+		}
+	}
+
+	core_analysis_bytes_size(core, core->block, core->blocksize, count);
+
+	if (obs != core->blocksize) {
+		rz_core_block_size(core, obs);
+	}
+	return RZ_CMD_STATUS_OK;
+}
+
+RZ_IPI RzCmdStatus rz_analyze_n_ins_esil_handler(RzCore *core, int argc, const char **argv) {
+	ut32 count = 1, obs = core->blocksize;
+	st32 l;
+
+	if (argc > 1) {
+		l = (st32)rz_num_math(core->num, argv[1]);
+		if (l <= 0) {
+			RZ_LOG_ERROR("Invalid zero or negative arguments.\n");
+			return RZ_CMD_STATUS_ERROR;
+		}
+		count = l;
+		l *= 8;
+		if (l > obs) {
+			rz_core_block_size(core, l);
+		}
+	}
+
+	core_analysis_bytes_esil(core, core->block, core->blocksize, count);
+
+	if (obs != core->blocksize) {
+		rz_core_block_size(core, obs);
+	}
+	return RZ_CMD_STATUS_OK;
+}
+
+RZ_IPI RzCmdStatus rz_analyze_opcode_handler(RzCore *core, int argc, const char **argv) {
+	int cur;
+	char *d;
+
+	if (argc < 2) {
+		cur = RZ_MAX(core->print->cur, 0);
+		core_analysis_bytes_desc(core, core->block + cur, core->blocksize, 1);
+	} else {
+		d = rz_asm_describe(core->rasm, argv[1]);
+		if (RZ_STR_ISEMPTY(d)) {
+			RZ_LOG_ERROR("Unknown mnemonic\n");
+			free(d);
+			return RZ_CMD_STATUS_ERROR;
+		}
+		rz_cons_println(d);
+		free(d);
+	}
+	return RZ_CMD_STATUS_OK;
+}
+
+RZ_IPI RzCmdStatus rz_display_opcode_handler(RzCore *core, int argc, const char **argv) {
+	sdb_foreach(core->rasm->pair, listOpDescriptions, core);
+	return RZ_CMD_STATUS_OK;
+}
+
+RZ_IPI RzCmdStatus rz_analyze_cycles_handler(RzCore *core, int argc, const char **argv) {
+	RzList *hooks;
+	RzListIter *iter;
+	RzAnalysisCycleHook *hook;
+	char *instr_tmp = NULL;
+	st32 ccl = 0;
+	RzConfigHold *hc = rz_config_hold_new(core->config);
+
+	rz_config_hold_i(hc, "asm.cmt.right", NULL);
+	rz_config_hold_i(hc, "asm.functions", NULL);
+	rz_config_hold_i(hc, "asm.lines", NULL);
+	rz_config_hold_i(hc, "asm.xrefs", NULL);
```

Comment:
```suggestion
	rz_config_hold_i(hc, "asm.cmt.right", "asm.functions", "asm.lines", "asm.xrefs", NULL);
```

not super important, JFYI

Reply:
Thanks for your comments. Already changed.